theseus/embodied/motionmodel/double_integrator.py

```diff
@@ -96,11 +101,13 @@ class GPCostWeight(CostWeight):
     def __init__(
         self,
         Qc_inv: Union[Variable, torch.Tensor],
-        dt: Union[Variable, torch.Tensor],
+        dt: Union[float, Variable, torch.Tensor],
         name: Optional[str] = None,
     ):
         super().__init__(name=name)
         if not isinstance(dt, Variable):
+            if not isinstance(dt, torch.Tensor):
+                dt = torch.tensor(dt)
```

Comment:
I have added a fix for this class in #157, make sure you merge those changes as well.